client/executor/src/native_executor.rs

```diff
@@ -267,6 +283,7 @@ impl<D: NativeExecutionDispatch> NativeExecutor<D> {
 			default_heap_pages,
 			host_functions,
 			max_runtime_instances,
+			None,
```

Comment:
Does this mean caching is disabled when running substrate node?

Reply:
Yes. I deliberately didn't put it here.

The thing is that this change is dictated by needs of polkadot. I believe it should mitigate the stalls we are seeing there right now. Therefore, it is critical to land it quickly. I figured that we should move the question about the cache out of the way.